src/keyboard-layout-manager-windows.cc

```diff
@@ -40,10 +40,12 @@ HKL GetForegroundWindowHKL() {
   return GetKeyboardLayout(dwThreadId);
 }
 
-KeyboardLayoutManager::KeyboardLayoutManager(v8::Isolate *isolate, Nan::Callback *callback) : isolate_(isolate), callback(callback) {}
+KeyboardLayoutManager::KeyboardLayoutManager(v8::Isolate *isolate, Nan::Callback *callback) : callback(callback) {
+  // callback is not used at all in windows, we are not detecting a change in keyboard layout
```

Comment:
If we're immediately deleting the callback, should we set the `callback` member variable to `nullptr` instead of leaving a dangling pointer?

Reply:
Done